src/main/java/com/impactupgrade/nucleus/service/segment/MailchimpCommunicationService.java

```diff
@@ -421,6 +424,12 @@ protected MemberInfo toMcMemberInfo(CrmContact crmContact, Map<String, Object> c
     mcContact.merge_fields.mapping.put(FIRST_NAME, crmContact.firstName);
     mcContact.merge_fields.mapping.put(LAST_NAME, crmContact.lastName);
     mcContact.merge_fields.mapping.put(PHONE_NUMBER, crmContact.mobilePhone);
+
+    mcContact.consents_to_one_to_one_messaging = true; //?
+    mcContact.sms_phone_number = crmContact.mobilePhone;
+    mcContact.merge_fields.mapping.put(SMS_PHONE_NUMBER, crmContact.mobilePhone);
+    mcContact.sms_subscription_status = SUBSCRIBED;
```

Comment:
We shouldn't set this (and shouldn't set the phone number or consents either) unless CrmContact has `smsOptIn=true` and `smsOptOut=false`

Reply:
Also, as mentioned in Slack, add a countryCode configuration in env.json's Mailchimp account section. If the contact's number doesn't start with that countryCode, skip opting it into SMS here -- it appears MC only allows numbers from the country the account's organization is within.